example-library/src/test/java/uk/gov/gchq/palisade/example/library/ResourceServicePrepopTest.java

```diff
@@ -37,14 +38,15 @@
 import uk.gov.gchq.palisade.service.resource.config.StdResourcePrepopulationFactory;
 import uk.gov.gchq.palisade.service.resource.stream.config.AkkaSystemConfig;
 
+import java.util.HashSet;
 import java.util.Map.Entry;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @EnableAutoConfiguration
-@ActiveProfiles({"example-k8s"})
+@ActiveProfiles({"example-test"})
```

Comment:
I don't see the yaml file for example-test.  Could you double check?

Reply:
The file is in the directory `example-library/src/test/resources`